eslint.config.ts

```diff
@@ -52,9 +52,23 @@ export default ts.config(
     rules: {
       'react/react-in-jsx-scope': 'off',
       'import-x/no-unresolved': 'off',
-      'import-x/no-named-as-default-member': 'off',
       '@typescript-eslint/consistent-type-imports': 'error',
       'react/prop-types': 'off',
+      'import-x/order': [
+        'error',
+        {
+          groups: [['builtin', 'external', 'internal']],
+          alphabetize: { order: 'asc', caseInsensitive: true },
+        },
+      ],
+      'prefer-const': 'error',
+      'no-var': 'error',
+      'import-x/named': 'error',
+      'import-x/namespace': 'error',
+      'import-x/default': 'error',
+      'import-x/export': 'error',
+      'import-x/no-named-as-default': 'error',
+      'func-style': ['error', 'expression', { allowArrowFunctions: true }],
```

Comment:
👍🏼

Reply:
I don't have much to consider, because we have https://eslint.org/docs/latest/rules/func-style
only `declaration` and `expression` and i prefer second, because it's more current, much fresh code using func as const instead of `functions` and i hope it's good to follow latest approaches in coding at all.

If it's satisfied you, let's resolve conversation, else feel free to ask more :)

Reply:
Sounds good. I was just curious, and I think it’s great for consistency. :)